config/clusters/catalystproject-africa/uvri.values.yaml

```
@@ -0,0 +1,70 @@
+jupyterhub:
+  ingress:
+    hosts: [uvri.af.catalystproject.2i2c.cloud]
+    tls:
+      - hosts: [uvri.af.catalystproject.2i2c.cloud]
+        secretName: https-auto-tls
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
```

This is lacking docs :( but let's also disable the configurator, per https://github.com/2i2c-org/infrastructure/issues/3349.
```suggestion
    jupyterhubConfigurator:
      enabled: false
    homepage:
      templateVars:
```
(I'll search for a place in the docs to add thi  so that is a bit more findable)